gst/nnstreamer/registerer/nnstreamer.c

```diff
@@ -95,6 +98,9 @@ gst_nnstreamer_init (GstPlugin * plugin)
   NNSTREAMER_INIT (plugin, transform, TRANSFORM);
   NNSTREAMER_INIT (plugin, if, IF);
   NNSTREAMER_INIT (plugin, rate, RATE);
+  NNSTREAMER_INIT (plugin, query_src, QUERY_SRC);
+  NNSTREAMER_INIT (plugin, query_sink, QUERY_SINK);
+  NNSTREAMER_INIT (plugin, query_filter, QUERY_FILTER);
```

Comment:
QUERY_FILTER may confuse users. I'd like to suggest
- QUERY_SERVER_SRC
- QUERY_SERVER_SINK
- QUERY_CLIENT

Reply:
Renamed elements based on your suggestion. PTAL